src/components/navigation/z-menu/index.tsx

```diff
@@ -147,7 +165,7 @@ export class ZMenu {
     return (
       <div class="menu-label">
         <div class="menu-label-content">
-          <slot></slot>
+          <slot onSlotchange={this.onLabelSlotChange}></slot>
```

Comment:
```suggestion
          <slot onSlotchange={this.onLabelSlotChange.bind(this)}></slot>
```

Reply:
already explained [here](https://github.com/ZanichelliEditore/design-system/pull/339#discussion_r1138343172) why I didn't put the `bind`